src/components/Banner/Banner.tsx

```diff
@@ -10,10 +10,11 @@ type Props = {
   label?: string
   error?: boolean
   children?: React.ReactNode
+  containerStyle?: ViewStyle
 }
 
-export const Banner = ({error, text, boldText, label, children}: Props) => (
-  <View style={[styles.banner, error === true && styles.bannerError]}>
+export const Banner = ({error, text, boldText, label, children, containerStyle}: Props) => (
```

Comment:
passing in styles for complex components is a smell

Reply:
definitely bad, reworking it.

Reply:
removed its usage